frontend/src/views/FieldAndSoil/SoilTests/SoilTests.tsx

```diff
@@ -59,69 +69,104 @@ export default function SoilTests({ fields, setFields }: FieldListProps) {
 
   return (
     <div>
-      <InfoBox>
-        Do you have soil test from within the past 3 years?
-        <ul>
-          <li>Yes - Select the lab used (soil test methods)</li>
-          <li>No - Click Next</li>
-        </ul>
-      </InfoBox>
-      <Dropdown
-        label="Lab (Soil Test Method)"
-        name="SoilTest"
-        value={soilTestData.SoilTest}
-        options={soilTestOptions}
-        onChange={handleChange}
-      />
-      {soilTestData.SoilTest !== '1' && (
-        <div>
-          {fields.map((field, index) => (
-            <ListItemContainer key={field.FieldName}>
-              <p>Field Name: {field.FieldName}</p>
-              <p>Sampling Month: {field.SoilTest.sampleDate}</p>
-              <p>NO3-N (ppm): {field.SoilTest.valNO3H}</p>
-              <p>P (ppm): {field.SoilTest.ValP}</p>
-              <p>K (ppm): {field.SoilTest.valK}</p>
-              <p>pH: {field.SoilTest.valPH}</p>
-              {Object.keys(field.SoilTest).length === 0 ? (
-                <button
-                  type="button"
-                  onClick={() => handleEditSoilTest(index)}
-                >
-                  Add Soil Test Results
-                </button>
-              ) : (
-                <>
-                  <button
-                    type="button"
-                    onClick={() => handleEditSoilTest(index)}
-                  >
-                    Edit Soil Test
-                  </button>
-                  <button
-                    type="button"
-                    onClick={() => handleDeleteSoilTest(index)}
-                  >
-                    Delete Soil Test
-                  </button>
-                </>
-              )}
-            </ListItemContainer>
-          ))}
-        </div>
-      )}
+      <ContentWrapper hasFields={fields.length > 0}>
+        {soilTestData.SoilTest === '1' && (
+          <InfoBox>
+            Do you have soil test from within the past 3 years?
+            <ul>
+              <li>Yes - Select the lab used (soil test methods)</li>
+              <li>No - Click Next</li>
+            </ul>
+          </InfoBox>
+        )}
+        <Dropdown
+          label="Lab (Soil Test Method)"
+          name="SoilTest"
+          value={soilTestData.SoilTest}
+          options={soilTestOptions}
+          onChange={handleChange}
+        />
+        {soilTestData.SoilTest !== '1' && (
```

Comment:
If one version is shown when soil test = 1 and a very different one is shown when it's another id, then these should be separate components.